baselines/Google Classroom Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -181,3 +182,37 @@ To configure the settings for Student Unenrollment:
 3.  Select **Student unenrollment**.
 4.  Select **Teachers Only**.
 5.  Select **Save**.
+
+## 5. Class Creation
+
+The first time users sign in to Classroom, they self-identify as either a student or teacher. Users who identify as teachers will be marked as a pending teacher until an administrator verifies them. Google Classroom allows administrators to restrict class creation to only verified teachers.
+
+### Policy
+
+#### GWS.CLASSROOM.5.1v0.2
+Who can create classes SHALL be set to verified teachers only.
```

Comment:
One final possible enhancement. The current policy wording heavily leverages the wording used in the UI "buttons," so it reads a little awkward as plain-English. If you do make this change, don't forget to also make the change in the drift rules.

```suggestion
Class creation SHALL be restricted to verified teachers only.
```